README.md

```diff
@@ -2,6 +2,20 @@
 
 # What first?
 
+## Types of VISAs
```

Comment:
Another thing that came to my mind: Maybe this can be in it's own section, something like `What first?` but for steps prior to move in